### PR TITLE
Add Bi-Dir FrSky Telem and YaapuScript to Upcoming Features

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Review by tridge — APPROVED** [review]

File: common/source/docs/common-yaapu-gcs.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+-----------------
+
+.. image:: ../../../images/Horus1.png
+
+
+
+Overview
+========
+
+In order to use this ability, you will need to setup the Yaapu LUAGCS script on your OpenTX transmitter. This script can be run con-currently with the Yaapu Telemetry script, but it is not required. Of course, the transmitter's display will only be used by one or the other at a time, but if the Telemetry script is running, important messages,warnings, and alerts will still be displayed while using the LUAGCS.
```
Comment (yaapu) [inline common/source/docs/common-yaapu-gcs.rst:31]: When the GCS script is running all other lua scripts are put on hold by OpenTX.
On the Horus the Lua GCS has telemetry info and status text messages and history (no warnings or sound alerts) while on the Taranis only battery voltage is displayed and there's no status text message history.

Comment (Hwurzburg) [reply common/source/docs/common-yaapu-gcs.rst:31]: thanks...will correct...otherwise,okay?

**Comment by Hwurzburg** [comment]
@yaapu can you approve now?

**Review by yaapu — APPROVED** [review]

**Comment by Hwurzburg** [comment]
thanks...merging

**Pull request merged**